packages/synapse-react-client/demo/containers/playground/ToastDemo.tsx

```diff
@@ -15,19 +15,28 @@ export const ToastDemo = () => {
   const [linkText, setLinkText] = React.useState('')
 
   function fireToast() {
+    const primaryButtonConfig =
+      buttonText === ''
+        ? undefined
+        : {
+            text: buttonText,
+            onClick: () => {
+              console.log('Primary button clicked!')
+            },
+          }
+    const secondaryButtonConfig =
+      linkText === ''
+        ? undefined
+        : {
+            text: linkText,
+            href: '#',
+          }
```

Comment:
In ToastDemo, `buttonText` and `linkText` are empty strings by default. `FullWidthAlert`, which is called by `displayToast`, expects the button config to be `undefined` when there shouldn't be a button or link added to the alert. Update the demo to pass `undefined` when the user hasn't entered a value for `buttonText` or `linkText`, so buttons aren't added to the toast